package/kernel/linux/modules/leds.mk

```diff
@@ -24,6 +24,24 @@ $(eval $(call KernelPackage,leds-gpio))
 
 LED_TRIGGER_DIR=$(LINUX_DIR)/drivers/leds/trigger
 
+define KernelPackage/led-group-multi-color
+  SUBMENU:=$(LEDS_MENU)
+  TITLE:=LEDs group multi-color support
+  KCONFIG:=CONFIG_LEDS_CLASS_MULTICOLOR \
+     CONFIG_LEDS_GROUP_MULTICOLOR
+  FILES:=$(LINUX_DIR)/drivers/leds/rgb/leds-group-multicolor.ko
```

Comment:
```
FILES:=$(LINUX_DIR)/drivers/leds/rgb/leds-group-multicolor.ko
     $(LINUX_DIR)/drivers/leds/led-class-multicolor.ko
```

Reply:
Would it make sense to add that as a separate module and add a depends ?

Reply:
`CONFIG_LEDS_CLASS_MULTICOLOR` is always activated except for the stm32 target.

I think it is best to fix the stm32 target to not activate it any more.

Reply:
You mean remove it by default in every platform and add it as a depends to every kmod that requires it?

Reply:
`CONFIG_LEDS_CLASS_MULTICOLOR` is already activated for all targets except `stm32`, see target/linux/generic/config-6.6.  Just Activate it for `stm32` too.